plugins/unshare.c

```diff
@@ -19,8 +19,15 @@ static rpmRC unshare_init(rpmPlugin plugin, rpmts ts)
 {
     char *paths = rpmExpand("%{?__transaction_unshare_paths}", NULL);
     private_mounts = argvSplitString(paths, ":", ARGV_SKIPEMPTY);
-    if (private_mounts)
-	unshare_flags |= CLONE_NEWNS;
+    if (private_mounts) {
+	/* Remounting "/" inside chroot fails with EINVAL */
```

Comment:
Turns out it's not really remounting (see my comment above), just setting propagation type on `/`.

Reply:
Yup, adjusted the comment.